open_humans/forms.py

```diff
@@ -4,15 +4,18 @@
 from django.conf import settings
 from django.core.mail import send_mail
 from django.template.loader import render_to_string
+from django.urls import reverse
 
 from allauth.account.forms import (ChangePasswordForm as AllauthChangePasswordForm,
                                    LoginForm as AllauthLoginForm,
                                    ResetPasswordForm as AllauthResetPasswordForm,
                                    SignupForm as AllauthSignupForm)
+from allauth.account.models import EmailAddress
 from allauth.account.utils import filter_users_by_email
 from allauth.socialaccount.forms import SignupForm as AllauthSocialSignupForm
 
-from common.utils import get_redirect_url
+from open_humans.settings import LOGIN_REDIRECT_URL
```

Comment:
probably should be like https://docs.djangoproject.com/en/2.1/topics/settings/#using-settings-in-python-code